apps/widget/src/components/notification-center/NotificationCenterWidget.tsx

```diff
@@ -1,6 +1,7 @@
 import { NotificationCenter, NovuProvider } from '@novu/notification-center';
+import { INovuThemeProvider } from '@novu/notification-center/dist/esm/types/store/novu-theme-provider.context';
```

Comment:
@venarius I think you can use `INovuTheme` from root component which is exported by the notification center:
```suggestion
import { INovuTheme } from '@novu/notification-center';
```